### ISSUE TITLE
Look for cases where `wlog` can simplify proofs.

### ISSUE THREAD

**Issue by MatthewDaggitt** [agda/agda-stdlib#523]
e.g. *-distribˡ-∣-∣

**Comment by MatthewDaggitt** [comment]
@gallais I had a go with this and can't seem to make it work for `*-distribˡ-∣-∣`. Care to have a go? Otherwise I suggest we probably close this as it's not exactly crucial.

**Comment by gallais** [comment]
We had the proof in the original PR thread: #450

**Comment by jamesmckinna** [comment]
So... five years later, what do we want to do about this?

**Comment by jamesmckinna** [comment]
Thanks @gallais for responding with #2577 ! 
Do you think that closes the issue, or would you want to keep it open after that gets merged?

**Comment by gallais** [comment]
I'm happy for it to be closed but there may still be further instances.

**Comment by jamesmckinna** [comment]
Indeed, tempting to think that the whole of `Data.Integer.Properties` might usefully be reviewed in this light, and `Data.Nat.Properties` (albeit to a lesser extent?). But maybe better to pick these things off piece by piece?

**Comment by jamesmckinna** [comment]
And... `wlog` really only operates on the symmetry induced by `flip` on operations/relations. 
What about other symmetries? cf. Läuchli (maybe this is/was (implicit?) in the original issue discussion/PR?)

**Comment by jamesmckinna** [comment]
It would be great if `wlog` could also be applied to the symmetry-heavy analyses in `Data.Nat.GCD`, but there the nature of 'wlog' cashes out as `Trichotomy`, with easy 'base' case when the two arguments are *equal*, and symmetry between the `_<_` and `_>_` cases... so not *obviously* applicable, sadly?

**Comment by jamesmckinna** [comment]
See also #2626 for more opportunities to use `wlog`... by making its type less constrained ;-)

**Comment by jamesmckinna** [comment]
My experience with trying to use `wlog` in #2971 suggests that we still don't really have the ergonomics of this combinator sorted out properly, when direct proof is so much less clunky...

**Comment by gallais** [comment]
Related work: https://msfp-workshop.github.io/msfp2026/submissions/msfp-final8.pdf
